adoc/admin-security-rbac.adoc

```diff
@@ -98,7 +98,7 @@ It is a known limitation of the addon customization process.
 Administrators can update the authentication connector settings after {productname}
 deployment as follows:
 
-. Based on the manifest in `~/clusters/<CLUSTER_NAME>/addons/dex/base/dex.yaml`, provide a kustomize patch to `~/clusters/<CLUSTER_NAME>/addons/dex/patches/dex-patch.yaml` of the form of strategic merge patch or a JSON 6902 patch.
+. Based on the manifest in `my-cluster/addons/dex/base/dex.yaml`, provide a kustomize patch to `my-cluster/addons/dex/patches/custom.yaml` of the form of strategic merge patch or a JSON 6902 patch.
```

Comment:
`my-cluster` here is a little confusing. I thought it is a configurable 'cluster name'

Reply:
All the doc use `my-cluster`. But I agreed it's better to use `<CLUSTER_NAME>`.
WDYT? @r0ckarong @nkoranova

Reply:
I need to check if this works properly. Our converter has some issues with using <> in some content types and maybe this needs to be escaped. Otherwise I'm OK with this change.

Reply:
I tried replacing all "my-cluster" mentions with <CLUSTER_NAME> and I didn't see a converter issue, but to me it feels slightly strange in come contexts. For example when my-cluster is the actual directory of the cluster and we tell the user to go into it: 
```
cd <CLUSTER_NAME> 
```
I think this requires a separate issue and PR.